packages/oas-to-har/__tests__/lib/style-formatting/index.test.js

```diff
@@ -0,0 +1,948 @@
+const undefinedArrayInput = [undefined];
+const objectInput = { R: 100, G: 200, B: 150 };
+const undefinedObjectInput = { R: undefined };
+
+const semicolon = ';'; // %3B when encoded, which we don't want
+const equals = '='; // %3D when encoded, which we don't want
+const comma = ','; // %2C when encoded, which we don't want
+
+/**
+ * These tests ensure that each style matches the spec: https://github.com/OAI/OpenAPI-Specification/blob/master/versions/3.1.0.md#style-values
+ *    and the examples https://github.com/OAI/OpenAPI-Specification/blob/master/versions/3.1.0.md#style-examples.
+ *
+ * Why 3.1.0 when it's not out yet, and we don't support it? Well all the prior documentation is riddled with documentation inconsistencies that appear
+ *    to be cleared up in 3.1.0. No new features, just less confusion.
+ *
+ * Sometimes we also refer to https://swagger.io/docs/specification/serialization for implementation clarification, such as when explode becomes multiple parameters, and when it stays as multiple values in one parameter
+ *
+ * I have copied the spec and example from the 3.1.0 docs below for easy reference
+ *
+ * Style Values
+ *
+ *   In order to support common ways of serializing simple parameters, a set of style values are defined.
+ *   style 	          type 	                    in 	            Comments
+ *   matrix 	        primitive, array, object 	path 	          Path-style parameters defined by RFC6570
+ *   label 	          primitive, array, object 	path 	          Label style parameters defined by RFC6570
+ *   form 	          primitive, array, object 	query, cookie 	Form style parameters defined by RFC6570. This option replaces collectionFormat with a csv (when explode is false) or multi (when explode is true) value from OpenAPI 2.0.
+ *   simple 	        array 	                  path, header 	  Simple style parameters defined by RFC6570. This option replaces collectionFormat with a csv value from OpenAPI 2.0.
+ *   spaceDelimited 	array, object 	          query 	        Space separated array or object values. This option replaces collectionFormat equal to ssv from OpenAPI 2.0.
+ *   pipeDelimited 	  array, object 	          query 	        Pipe separated array or object values. This option replaces collectionFormat equal to pipes from OpenAPI 2.0.
+ *   deepObject 	    object 	                  query 	        Provides a simple way of rendering nested objects using form parameters.
+ *
+ *
+ *  Style Examples
+ *
+ *  Assume a parameter named color has one of the following values:
+ *
+ *     string -> "blue"
+ *     array -> ["blue","black","brown"]
+ *     object -> { "R": 100, "G": 200, "B": 150 }
+ *
+ *  The following table shows examples of rendering differences for each value. (NON-SPEC-NOTE: I've added the additional `in` column for easy reference)
+ *  style 	        explode 	empty 	string 	      array 	                              object                                  in
```

Comment:
Can we tidy up the whitespace on this comment so we can see the alignment a little clearer?

Reply:
looks like there's a weird mix of spaces and tabs. fixing.